Handreiking API Design/Handreiking APi Design.md

```diff
@@ -16,24 +18,31 @@ Dit document heeft een tweeledig doel.
 
 **Motivatie en dominante domeinen**
 
-Bij relaties tussen twee domeinen wil je vanuit elk domein de gerelateerde
-gegevens uit het andere domein kunnen opvragen/inkijken. Eén van de domeinen
-wordt als dominant/primair domein beschouwt. Soms is heel duidelijk welke dat
+Bij relaties tussen twee domeinen wil je vanuit beide domeinen de gerelateerde
+gegevens uit het andere domein kunnen opvragen/inkijken. Daarvoor wordt in elk domein een relatie-entiteit gedefinieerd die in feite de representatie van een entiteit in een ander domein is. Zo'n relatie-entiteit kan vervolgens gebruikt worden om aan andere entiteiten binnen het domein te koppelen. Dit is noodzakelijk omdat de relatie tussen de entiteit in het ene domein en de entiteit in het andere domein slechts op logisch niveau kan worden gelegd. Zo'n relatie wordt nooit op technisch niveau gecreëerd. 
+
+In het voorbeeld hieronder zie je dat in het 'Zaken API' domein de relatie entiteit 'zaakverzoek' is gecreëerd wat in feite een representatie is van de entiteit 'verzoek' uit het 'Klantinteracties API' domein. In het 'Klantinteracties API' domein is juist de relatie-entiteit 'objectverzoek' gecreëerd wat een representatie is van de entiteit 'zaak' uit het 'Zaken API' domein.
+De relatie tussen 'zaakverzoek' en 'verzoek' bestaat alleen op logisch niveau en wordt op technisch niveau niet gerealiseerd. Hetzelfde geldt voor de relatie tussen 'objectverzoek' en 'zaak'.
+
+![Voorbeeld relatie-entiteiten](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/Handreiking%20API%20Design/Relatie-entiteiten.jpg)
+
+Binnen een relatie tussen domeinen wordt altijd één van de domeinen
+als dominant/primair beschouwt. Soms is heel duidelijk welke dat
 is en is dat goed te beargumenteren, andere keren is het een buikgevoel en
 lastiger te beargumenteren, of het is gewoon een keuze omdat er geen
-natuurlijke dominantie kan aangewezen worden. Voorbeelden hiervan zijn:
+natuurlijke dominantie aangewezen kan worden. Voorbeelden hiervan zijn:
 
 (dominante domein in vet)
 
-* **Zaken** en informatieobjecten
-* **Zaken** en klaninteracties
-* **Besluiten** en informatieobjecten
+* **Zaken** en Documenten
```

Comment:
```suggestion
* **Zaken** en Informatieobjecten
```

Reply:
We kennen het type Document niet meer, dat is Informatieobject geworden. Als we dat in documentatie weer documenten gaan noemen is dat verwarrend, niemand kan nl. klasse Document terug vinden in de API's

Reply:
Helemaal gelijk.